manifests/plugin/pcie_errors.pp

```diff
@@ -0,0 +1,15 @@
+class collectd::plugin::pcie_errors (
```

Comment:
can you please add puppet-string documentation for the new parameters?

Reply:
I added documentation for the parameters and changed Boolean ones to not be optional.